doc/content/resources/multiapps_chps/chp_4_workflow.md

```diff
@@ -0,0 +1,118 @@
+
+If child applications are established at multiple positions, field data of the parent app may need to be sampled and transferred to each child application (usually as scalar data), or vice versa. Based on the different type of field data and different algorithm, [`MultiAppMeshFunctionTransfer`](https://mooseframework.inl.gov/source/transfers/MultiAppMeshFunctionTransfer.html)(+both directions+), [`MultiAppPostprocessorInterpolationTransfer`](https://mooseframework.inl.gov/source/transfers/MultiAppPostprocessorInterpolationTransfer.html)(+from child application only+), [`MultiAppUserObjectTransfer`](https://mooseframework.inl.gov/source/transfers/MultiAppUserObjectTransfer.html)(+to child application only+), [`MultiAppVariableValueSamplePostprocessorTransfer`](https://mooseframework.inl.gov/source/transfers/MultiAppVariableValueSamplePostprocessorTransfer.html)(+to child application only+), and [`MultiAppVariableValueSampleTransfer`](https://mooseframework.inl.gov/source/transfers/MultiAppVariableValueSampleTransfer.html)(+to child application only+).
+
+For the SFR example, [wf_step_6] visualizes how the five apps involved are tested. The field variable `power_density` calculated by Griffin is transferred to Bison thermal physics simulation using `MultiAppProjectionTransfer` as this transfer conserves the integrated power, leveraging $L^2$ projection to minimize difference between the integrated power. For the temperature variable from heat conduction and thermal hydraulics applications, as well as the displacement variables from tensor mechanics application, `MultiAppInterpolationTransfer` is used to ensure proper values can be obtained when meshes are different. Because the thermal physics application and the tensor mechanics applications use an identical mesh, field variables such as temperature and displacements are transferred by `MultiAppCopyTransfer`. The data transfer between the thermal physics application (BISON) and the thermal hydraulics (SAM) application is special. Although both applications use 2D-RZ axisymmetric meshes, the applications are oriented in different coordinate systems. In BISON, the convention is that the R mesh exists in the XZ plane, while in SAM, the convention is that R mesh exists in the XY plane. `MultiAppCoordSwitchNearestNodeTransfer` (a SAM object) performs transfers with coordinate system conversion along with the nearest node transfer. Native support is also present in MOOSE for most coordinate-transformations during transfers.
```

Comment:
to conserve power, one has to use the two postprocessors. 

a L2 projection in general will not conserve power (L1 norm), unless strict conditions are met

I would cut this paragraph in sections for each transfer for readability
also add a code listing in between  each section